management/dns_update.py

```diff
@@ -966,19 +966,23 @@ def set_secondary_dns(hostnames, env):
 				# Resolve hostname.
 				try:
 					response = resolver.query(item, "A")
+				except (dns.resolver.NoNameservers, dns.resolver.NXDOMAIN, dns.resolver.NoAnswer):
+					response = resolver.query(item, "AAAA")
```

Comment:
I believe this will cause the control panel to crash on an unresolvable domain.

Reply:
Sigh, tfw your python is rusty 😂

I guess we really need to nest `try` statements. :(